metabase/metabase
# Add Select all in ListValuePicker in the QB

Epic https://github.com/metabase/metabase/issues/47860
Closes https://github.com/metabase/metabase/issues/47861
Slack discussion https://metaboat.slack.com/archives/C0645JP1W81/p1726078869890019

How to verify:
- New -> Question -> Orders
- Filter -> Product -> Category
- There should be new `Select all` option which changes to `Select none` if all items are selected

<img width="483" alt="Screenshot 2024-09-11 at 15 29 22" src="https://github.com/user-attachments/assets/7c5011de-5c75-4463-84ce-04c3bdeda9f5">
<img width="480" alt="Screenshot 2024-09-11 at 15 29 33" src="https://github.com/user-attachments/assets/24813d29-cc74-41f6-b6ca-aeb002d2ecac">
<img width="476" alt="Screenshot 2024-09-11 at 15 29 37" src="https://github.com/user-attachments/assets/1d3b71bb-6d87-4b1b-be8b-0c7c548102c4">
